tests/standalone_activity_test.go

```diff
@@ -897,6 +905,9 @@ func (s *standaloneActivityTestSuite) TestStartToCloseTimeout() {
 				Name: taskQueue.Name,
 			},
 			StartToCloseTimeout: durationpb.New(1 * time.Second),
+			RetryPolicy: &commonpb.RetryPolicy{
+				MaximumAttempts: 1,
+			},
```

Comment:
https://github.com/temporalio/temporal/pull/8723 fixed things so that we _do_ schedule retries when `schedule-to-close` is missing. But this test was expecting that the activity would fail on a `start-to-close` timeout. Hence it needed to be changed to limit to one attempt. This was actually broken on  pre-merge `standalone-activity`.